input/input.css

```diff
@@ -0,0 +1,131 @@
+@import url(/style.css);
+
+.input-intro {
+  font-size: 1.25rem;
+  padding: 1rem 1rem;
+}
+.form-heading {
+  margin: 5px 0;
+  color: var(--darker-grey);
+}
+
+.form-login {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  width: 16rem;
+  height: 20rem;
+  border-radius: 10px;
+  background-color: var(--off-white);
+  margin-bottom: 1rem;
+}
+.form-label,
+.password-label {
+  font-size: 0.875rem;
+}
+.input-container,
+.error-input {
+  padding: 0 10px;
+  display: flex;
+  flex-direction: column;
+  gap: 7px;
+}
+.form-input,
+.error-for-password-input {
+  padding: 5px;
+  width: 13rem;
```

Comment:
same here, try to maintain same units

Reply:
it is less than 10px so gave values in px here